### PR TITLE
Create IFCTunnelDep2.1.ifc

### PR DESCRIPTION
**Pull request by larswik**
## File

Example file for sprint 2.1

## Covered Concept Templates - Usages Included

All templates/usages for sprints 1.1, 1.2, 1.3 and 2.1

## Miscellaneous

Alignment layout created by LandXML - IFC Converter from RDF Ltd. (Peter Bonsma).

GeometryGym library used to create the main structure of the file.

##

### CODE REVIEW

File: files/project_team/IFCTunnelDep2.1.ifc [hunk 1]
```diff
@@ -0,0 +1,604 @@
...
+#79= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#80));
+#80= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#81,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCPARAMETERVALUE(0.0213755629473684),#84);
+#81= IFCAXIS2PLACEMENT2D(#82,#83);
+#82= IFCCARTESIANPOINT((160980.931044,6721776.117138));
+#83= IFCDIRECTION((-0.99210406,-0.12541741));
+#84= IFCCIRCLE(#85,1900.0);
+#85= IFCAXIS2PLACEMENT2D(#86,#87);
+#86= IFCCARTESIANPOINT((238.293085,-1884.99772));
+#87= IFCDIRECTION((-0.12541741,0.99210406));
+#88= IFCALIGNMENTHORIZONTALSEGMENT(', code=SC',', code=CS',#89,-3.01584409831919,1900.0,1900.0,40.61357,$,.CIRCULARARC.);
```
Comment (pjanck) [inline files/project_team/IFCTunnelDep2.1.ifc:104]: Interesting tags :)

File: files/project_team/IFCTunnelDep2.1.ifc [hunk 2]
```diff
@@ -0,0 +1,604 @@
...
+#252= IFCLOCALPLACEMENT($,#253);
+#253= IFCAXIS2PLACEMENT3D(#254,$,$);
+#254= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#255= IFCPRODUCTDEFINITIONSHAPE($,$,(#256));
+#256= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#257));
+#257= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#258,IFCPARAMETERVALUE(1.5642963819142),IFCPARAMETERVALUE(-0.002299793167857),#261);
+#258= IFCAXIS2PLACEMENT2D(#259,#260);
+#259= IFCCARTESIANPOINT((28014.368413,91.482242));
+#260= IFCDIRECTION((0.99997888,-0.0064999));
+#261= IFCCIRCLE(#262,28000.0);
```
Comment (pjanck) [inline files/project_team/IFCTunnelDep2.1.ifc:277]: 28km for a radius is quite big. Please double-check.

**Review by pjanck — APPROVED** [review]

File: files/project_team/IFCTunnelDep2.1.ifc (hunk 2 above)
Comment (larswik) [reply files/project_team/IFCTunnelDep2.1.ifc:277]: They must use big radius:es in the vertical alignment for rail. From the LandXML:
         <CircCurve radius="-10000.0000000" length="185.8518247">
              27050.062737 84.264410
          </CircCurve>
          <CircCurve radius="17000.0000000" length="127.4875501">
              27675.235592 91.453897
          </CircCurve>
          <CircCurve radius="11000.0000000" length="115.4981945">
              27887.874451 92.304453
          </CircCurve>
          <CircCurve radius="28000.0000000" length="64.3942087">
              28046.564851 91.272965
          </CircCurve>
          <CircCurve radius="-25000.0000000" length="25.0052981">
              28535.289674 86.972187
          </CircCurve>
          <CircCurve radius="20000.0000000" length="22.0040455">
              29007.234710 83.291165
          </CircCurve>

File: files/project_team/IFCTunnelDep2.1.ifc (hunk 1 above)
Comment (larswik) [reply files/project_team/IFCTunnelDep2.1.ifc:104]: From the LandXML. A roundtrip LandXML=>IFC=>LandXML was tested.

File: files/project_team/IFCTunnelDep2.1.ifc (hunk 2 above)
Comment (pjanck) [reply files/project_team/IFCTunnelDep2.1.ifc:277]: I see. Thank you for the snippets!

**Pull request merged**